back/engines/commercial/insights/spec/models/insights/view_spec.rb

```diff
@@ -11,18 +11,17 @@
 
   describe 'validations' do
     it { is_expected.to validate_presence_of(:name) }
-    it { is_expected.to validate_presence_of(:scope) }
     it { is_expected.to validate_uniqueness_of(:name) }
   end
 
   describe 'associations' do
+    it { is_expected.to have_many(:data_sources).dependent(:destroy) }
```

Comment:
Only this line is new. I just replaced `expect(view)` by `is_expected` in the others.